### ISSUE TITLE
[Feature] Language Filter lock

### ISSUE THREAD

**Issue by GeneralUser-NonDev** [manga-download/hakuneko#6365]
**Is your feature request related to a problem? Please describe.**

When selecting a title the filter always, annoyingly, reset.

**Describe the solution you'd like**

Either a padlock icon to lock/unlock the filter next to the filter drop-down arrow or a static selection for a specific filter that doesn't reset upon selecting the next title from the connector/bookmark list.  (ex: English)

**Describe alternatives you've considered**

Maybe a field in the settings to lock the filter to a specific language

**Context (please complete the following information):**
 - Your current HakuNeko Version [e.g. 6.17]

Latest Nightly (master@4a29b4)

**Additional context**

Filter Locked
![Hakuneko FilterLock](https://github.com/manga-download/hakuneko/assets/96662916/3bc4f12c-ff37-4901-b139-f8ab3f08bba1)
Filter Unlocked
![Hakuneko FilterUnLock](https://github.com/manga-download/hakuneko/assets/96662916/98af3019-df86-41db-b649-7b1a4e2d2b6d)

**Comment by Sheepux** [comment]
Yup,title language storage is planned in haru (I've put the button already but it's doing nothing yet)
